src/components/app/userActionViewKeyPageDeeplinkWrapper/gb.tsx

```diff
@@ -0,0 +1,30 @@
+import { SearchParams } from 'next/dist/server/request/search-params'
+
+import {
+  CampaignMetadataMap,
+  UserActionViewKeyPageDeeplinkWrapper,
+} from '@/components/app/userActionViewKeyPageDeeplinkWrapper'
+import { SupportedCountryCodes } from '@/utils/shared/supportedCountries'
+import { GBUserActionViewKeyPageCampaignName } from '@/utils/shared/userActionCampaigns/gb/gbUserActionCampaigns'
+
+const countryCode = SupportedCountryCodes.GB
+
+const campaignMetadataMap: CampaignMetadataMap<keyof typeof GBUserActionViewKeyPageCampaignName> = {
+  NEWMODE_EMAIL_ACTION: {
```

Comment:
This should use the campaign enum, not a hardcoded value
```suggestion
  [GBUserActionViewKeyPageCampaignName.NEWMODE_EMAIL_ACTION]: {
```

In a lot of cases the key might not equal the value exactly, would not be consistent with other maps throughout the app